<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingStatisticsChartClass</class>
 <widget class="QMainWindow" name="TrainingStatisticsChartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainingStatisticsChart</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="100,1">
    <item>
     <widget class="QCustomPlot" name="ChartDisplayWidget" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="iteration_checkBox">
        <property name="font">
         <font>
          <family>Cascadia Mono SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>iteration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fAvgLoss_checkBox">
        <property name="font">
         <font>
          <family>Cascadia Mono SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>fAvgLoss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fPosAcc_checkBox">
        <property name="font">
         <font>
          <family>Cascadia Mono SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>fPosAcc</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
